sdk/tracing/azotel/otel.go

```diff
@@ -14,13 +14,16 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore/tracing"
 	"go.opentelemetry.io/otel/attribute"
 	"go.opentelemetry.io/otel/codes"
+	"go.opentelemetry.io/otel/propagation"
 	semconv "go.opentelemetry.io/otel/semconv/v1.17.0"
 	"go.opentelemetry.io/otel/trace"
 )
 
 // TracingProviderOptions contains the optional values for NewTracingProvider.
 type TracingProviderOptions struct {
-	// for future expansion
+	// propagator is the OpenTelemetry Propagator to use.
+	// If not set, a propagation.TraceContext Propagator is used.
+	propagator propagation.TextMapPropagator
```

Comment:
This will need to be exported so that callers can set it.

What's the default behavior of `propagation.TraceContext`, is it a no-op propagator?

Reply:
Oops, thanks for catching that!

[`propagation.TraceContext`(https://github.com/open-telemetry/opentelemetry-go/blob/main/propagation/trace_context.go) is OTel's implementation of W3C trace context propagator. 
 
From our offline chat, @lmolkova mentioned OTel gives user an ability to customize propagator, but we (at least for now) ignore it in other languages and use the W3C trace context implementation. Therefore we are setting it as the default.

Reply:
If we ignore it in other languages, then do we even need to expose this (we can always add it later)?

Reply:
I agree that we don't need to expose it until there is a need to support custom propagators (which should then be done across all languages).